taccsite_cms/locale/en/LC_MESSAGES/django.po

```diff
@@ -53,12 +53,6 @@ msgstr ""
 "Users can't move a page without permissions to change the page. "
 "Ensure user has \"Can edit\" permission."
 
-
-#: djangocms-blog
-#: templates/djangocms_blog/includes/blog_meta.html:6
-msgid "by"
-msgstr "by:"
-
```

Comment:
**But… but we still use `trans "by"` in Core-CMS `blog_meta.html`… ?!** Yes. This deletion just removes our custom text for it, and uses "by" as is.